examples/asr/asr_chunked_inference/rnnt/speech_to_text_buffered_infer_rnnt.py

```diff
@@ -194,8 +215,29 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
             decoding_cfg.strategy = "greedy_batch"
         decoding_cfg.preserve_alignments = True  # required to compute the middle token for transducers.
         decoding_cfg.fused_batch_size = -1  # temporarily stop fused batch during inference.
-
-    asr_model.change_decoding_strategy(decoding_cfg)
+        decoding_cfg.beam.return_best_hypothesis = True  # return and write the best hypothsis only
+
+    # Setup decoding strategy
+    if hasattr(asr_model, 'change_decoding_strategy'):
+        if not isinstance(asr_model, EncDecRNNTModel) and not isinstance(asr_model, EncDecHybridRNNTCTCModel):
+            raise ValueError(
+                "The script supports rnnt model and hybrid model with rnnt decodng! use ctc/speech_to_text_buffered_infer_ctc.py for other conditions."
+            )
+        else:
+            # rnnt model
+            if isinstance(asr_model, EncDecRNNTModel):
+                asr_model.change_decoding_strategy(decoding_cfg)
+
+            # hybrid ctc rnnt model with decoder_type = rnnt
+            if isinstance(asr_model, EncDecHybridRNNTCTCModel):
```

Comment:
If we drop the decoder_type from the config, we can assume that it is always rnnt. So we would call the change_decoding_strategy by decoder_type="rnnt".